optimizely/notification/center.go

```diff
@@ -49,6 +50,16 @@ func (c *DefaultCenter) AddHandler(notificationType Type, handler func(interface
 	return -1, fmt.Errorf("no notification manager found for type %s", notificationType)
 }
 
+// RemoveHandler removes a handler for the given id and notification type
+func (c *DefaultCenter) RemoveHandler(id int, notificationType Type) error {
```

Comment:
if you have a better name for RemoveHandler, then please use it. we have that name on manager already. (I am not really enforcing - just suggesting). otherwise lgtm

Reply:
thanks for the feedback, any suggestions?

Reply:
We also have AddHandler method on both notificationCenter and manager, we will have to change that aswell then?

Reply:
to remove confusion, I would probably remove "Handler" word in notification package. I would have just Add and Remove. Since this is notification package, we know we are adding and removing a notification.  But like I said, I will not enforce it. I have just seen a lot of cases of overusing methods with same name for different objects.

Reply:
i think so too that `Add` and `Remove` make more sense, have made these changes, can you please review it now?

Reply:
are we adding a notification, or a decision ? I think you did the other way around

Reply:
we are adding notification listener for any notification type, but currently we have only `Decision` type as of now in go. I have updated the method definition for notification-manager now instead of notification-center. Can you please re-review it?